clusterloader2/pkg/measurement/common/api_availability_measurement.go

```diff
@@ -210,6 +213,11 @@ func (a *apiAvailabilityMeasurement) initFields(config *measurement.Config) erro
 			return err
 		}
 		a.hostPollTimeoutSeconds = hostPollTimeoutSeconds
+		hostPollExecTimeoutSeconds, err := util.GetIntOrDefault(config.Params, "hostPollExecTimeoutSeconds", 10)
```

Comment:
How about using util.GetDurationOrDefault?

This will allow us to drop "seconds" suffix, as it will be clear from the context that the unit is what you specified in the config (e.g. "1s", "10m") and allow to drop seconds to time.Duration convertion in line 99.

Reply:
I just wanted to be consistent with hostPollTimeoutSeconds. But I think we can change both of them tbh